jupyter_server/serverapp.py

```diff
@@ -2540,6 +2540,10 @@ def load_server_extensions(self) -> None:
         """
         self.extension_manager.load_all_extensions()
 
+    # is this how I would expose it? and Is this a good name?
```

Comment:
Yep, this looks good to me. Let's add a docstring here to explain what is returned here.